src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -480,7 +480,7 @@ private function updateSchema(): void
         $comparator = $this->createComparator($schemaManager);
         $schemaDiff = $this->compareSchemas($comparator, method_exists($schemaManager, 'introspectSchema') ? $schemaManager->introspectSchema() : $schemaManager->createSchema(), $this->getSchema());
         $platform = $this->driverConnection->getDatabasePlatform();
-        $queries = method_exists($platform, 'getAlterSchemaSQL') ? $platform->getAlterSchemaSQL($schemaDiff) : $schemaDiff->toSaveSql($platform);
+        $queries = $schemaDiff->toSaveSql($platform);
```

Comment:
But toSaveSql is deprecated, isn't it? so we need to find the future-proof version of this.

Reply:
Yeah, that's true, but I haven't found an adequate replacement yet 🥲 Therefore I'd keep the current implementation until there's a solution offered by Doctrine. 
Probably `doctrine/dbal` needs to add a parameter to `getAlterSchemaSQL()` or offers an additional method for that 🤔